src/Compilers/Core/AnalyzerDriver/DeclarationComputer.cs

```diff
@@ -13,9 +13,7 @@ internal class DeclarationComputer
         internal static DeclarationInfo GetDeclarationInfo(SemanticModel model, SyntaxNode node, bool getSymbol, IEnumerable<SyntaxNode> executableCodeBlocks, CancellationToken cancellationToken)
         {
             var declaredSymbol = getSymbol ? model.GetDeclaredSymbol(node, cancellationToken) : null;
-            var codeBlocks = executableCodeBlocks == null ?
-                ImmutableArray<SyntaxNode>.Empty :
-                executableCodeBlocks.Where(c => c != null).AsImmutableOrEmpty();
+            var codeBlocks = executableCodeBlocks?.Where(c => c != null).AsImmutableOrEmpty() ?? ImmutableArray<SyntaxNode>.Empty;
```

Comment:
Maybe others disagree but I don't find this new code to be more readable.  It takes a simple ternary and replaces it with a ?. and a ??.